source/globalCommands.py

```diff
@@ -898,6 +898,31 @@ def script_toggleReportClickable(self,gesture):
 			config.conf["documentFormatting"]["reportClickable"]=True
 		ui.message(state)
 
+	@script(
+		description=_(
+			# Translators: Input help mode message for cycle through automatic language switching mode command.
+			"Cycles through speech automatic language switching modes off, language only and language and dialect."

```

Comment:
I found this sentence hard to parse. Alternative suggestion (adds one word):

```suggestion
			"Cycles through speech modes for automatic language switching: off, language only and language and dialect."
```

Reply:
Thanks. Fixed in cfa51bafe7a460066828d19c0648401c3b1bca21.